tests/users/test_queries.py

```diff
@@ -0,0 +1,117 @@
+import pytest
+import uuid
+from graphql_relay import to_global_id
+
+from django.contrib.auth import get_user_model
+from creator.organizations.factories import OrganizationFactory
+from creator.organizations.models import Organization
+from creator.users.factories import UserFactory
+
+User = get_user_model()
+
+ALL_USERS_ORGANIZATION = """
+query ($organizations: [ID]) {
```

Comment:
can we make this type `$organizations: [ID!]`? If you add a value to the list we shouldn't allowed it to be `null`